experiment/dispatcher.py

```diff
@@ -118,7 +122,7 @@ def dispatcher_main():
     scheduler_loop_thread = threading.Thread(target=scheduler.schedule_loop,
                                              args=(experiment.config,))
     scheduler_loop_thread.start()
-    measurer_loop_thread = threading.Thread(
+    measurer_loop_thread = multiprocessing.Process(
```

Comment:
When running locally, will this schedule all the trials at once in parallel?

Reply:
Yes. We can add something to limit this in a later iteration.